### ISSUE TITLE
Feature request: output search results after protein inference and filtering

### ISSUE THREAD

**Issue by hendrikweisser** [nf-core/quantms#27]
### Description of feature

In the DDA/label-free workflow (perhaps in others as well), it would be useful to get an output of the identification results at the end of the search/inference/filtering pipeline, but before quantification. This would help for QC-purposes and to get a fuller understanding of the data on the ID level.

Currently the latest "ID-only" output is in the "idfilter" directory, containing results after database search, rescoring, (consensus ID, if applicable) and filtering according to "psm_pep_fdr_cutoff" - but before protein inference and protein-level filtering ("protein_level_fdr_cutoff"). Later outputs (after ProteomicsLFQ) only contain results for quantified IDs - omitting those that were identified, but never quantified. (Is that correct?)

**Comment by jpfeuffer** [comment]
Yes, correct. However, this would need to be done inside ProteomicsLFQ. So you would need to raise an issue at OpenMS and link it here.
